public/themes/modern.css

```diff
@@ -351,15 +351,14 @@ td.itdc {
 div.sni {
 	text-align: center;
 	margin: 2px auto 6px;
-	padding: 0 10px 5px;
+	padding: 0 0px 5px;
```

Comment:
Changes in this file haven't been removed from the PR, it seems. 🙂  
(The help text also hasn't been updated to mention F for fullscreen 🙏)